FetchingEmailContents.cls

```diff
@@ -0,0 +1,106 @@
+/**
+* @description: Email Searvice to get the email from the user which consist of 
+                search query in the email body.If the user contact does not exist,
+                create new contact record and if the user query is new create new
+                email_handler__c record.As there is juction between contact and 
+                email_handler__c new record must be created in it.
+* @author: Noopur Naik
+*
+* @revision:
+* Version      Date            Description
+* ------------------------------------------------
+* 1.0          28-Jun-2017     Initial revision.
+**/
+
+
+global class FetchingEmailContents implements Messaging.InboundEmailHandler {
+    global  String emailHandlerid;
+    global  String contactId;
+    global Messaging.InboundEmailResult handleInboundEmail(Messaging.InboundEmail email, Messaging.InboundEnvelope envelope) {
+    Messaging.InboundEmailResult result = new Messaging.InboundEmailresult();
+     String contactId;
+    
+     List<String> contactName =  email.fromname.split(' ');
+     String contactEmail = envelope.fromaddress;
+     String emailSubject = email.subject;
+     String emailBody = email.plainTextBody;
+     
+     List<Contact> contacts = [SELECT Id
```

Comment:
if you are checking count only then query should contain `count` function only. No need to select for Id